### PR TITLE
Add field "trainer's notes"

### PR DESCRIPTION
**Pull request by saisilinus**
This feature gives users the option to add personal notes to each resource/module

Fixes ASKnetCommunity/Training#72

### CODE REVIEW

**Comment by saisilinus** [comment]
Hi @bmen. I need your help with updating the styling files. Which files should I update? the sass or CSS files? I can't get my updates to work.

**Comment by timmwille** [comment]
> Hi @bmen. I need your help with updating the styling files. Which files should I update? the sass or CSS files? I can't get my updates to work.

is this still a valid question? I might also need a bit more time for review, as this is quite a lot of code to go through and will ask @bmen for some help here too

**Comment by saisilinus** [comment]
> > Hi @bmen. I need your help with updating the styling files. Which files should I update? the sass or CSS files? I can't get my updates to work.
> 
> is this still a valid question? I might also need a bit more time for review, as this is quite a lot of code to go through and will ask @bmen for some help here too

I used a workaround so it's not an issue. It would be nice to have the question answered though.

File: training.html [hunk 1]
```diff
@@ -79,6 +96,8 @@ <h3><i class="fas fa-filter"></i> Filter modules by tags</h3>
                     <h2>{{ module.module.name }}</h2>
                     <p class="intro">Learning Objective:</p>
                     {% if module.module.description %}<p class="intro"> {{ module.module.description | markdownify }}</p>{% endif %}
+                    <p class="trainer-notes-display"></p>
+                    <a class="edit-trainer-notes-button edit intro">Add additional notes</a>
```
Comment (timmwille) [inline training.html:100]:
```suggestion
                    <a class="edit-trainer-notes-button edit intro">Add trainer notes</a>
```

**Review by timmwille — APPROVED** [review]
small changes but everything else looks great, thx

File: assets/js/index.js [hunk 2]
```diff
@@ -779,4 +823,5 @@ window.onload = function () {
     initiateMobileButtons();
     calculateTime();
     calculateSummary();
+    initiateEditNotes();
```
[the remark this answers is not in the document]
Comment (saisilinus) [reply assets/js/index.js:841]: Yes. It will actually resolve a merge conflict

File: training.html (hunk 1 above)
Comment (saisilinus) [reply training.html:100]: This is okay too

File: assets/js/index.js [hunk 3]
```diff
@@ -779,4 +838,5 @@ window.onload = function () {
...
```
Comment (timmwille) [inline assets/js/index.js:841]:
```suggestion
```

**Review by timmwille — APPROVED** [review]
all good, sorry I think I can do the rest myself and merge.

**Pull request merged**